CHANGELOG.md

```diff
@@ -29,6 +29,7 @@
 - ScaledJob: introduce `RolloutStrategy` ([#2164](https://github.com/kedacore/keda/pull/2164))
 - Add ScalersCache to reuse scalers unless they need changing ([#2187](https://github.com/kedacore/keda/pull/2187))
 - Cache metric names provided by KEDA Metrics Server ([#2279](https://github.com/kedacore/keda/pull/2279))
+- Return minimum value for the metric and log, when cloudwatch returns empty list of metrics ([#2345](https://github.com/kedacore/keda/pull/2345))
```

Comment:
Maybe should this be placed under Improvements? WDYT @zroubalik ?

Reply:
Yeah and please put it below the existing line with AWS Cloudwatch Scaler entry. And pls try to follow the same syntax of the changelog message

Reply:
Done